index.html

```diff
@@ -0,0 +1,23 @@
+
+
+<h1 id = "title"> 
+    PEDRO JOAQUIN CHAMORRO CARDENAL
+    </h1>
+  <br>
+  
+  <Main id = "main">
+  <link rel="stylesheet" href="styles.css">
+      <div id= "img-div" div> 
+        <img id = "image"> <img src ="https://www.confidencial.digital/wp-content/uploads/2022/01/PEDRO-JOAQUIN-CHAMORRO1.jpg.webp"> <figcaption id ="img-caption" >Last picture taken to Pedro Joaquin Chamorro </figcaption>
+  <head> 
+    <div id="tribute-info">
+    <ul>
+   <li>   Pedro Joaquin Chamorro exemplified the values and principles of full democracy and free elections: political pluralism and accountability; no successive terms; separation of public from private interests; the struggle against corruption; reforms with social justice; freedom of the press.</li>
+    
```

Comment:
[OPTIONAL] It's very good in requesting this change, however, it will be very nice if you could add emojis to your inline comments. Emojis can enhance communication by adding visual cues to your comments. They can help convey emotions, tone, or intent that may not be immediately apparent from text alone. For example, using a thumbs-up emoji can indicate approval or agreement, while a confused face emoji can indicate that something is unclear. :slightly_smiling_face: